F21/gantlavamshikrishnareddy/vamshi_5030_03.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""Vamshi-5030-03.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1eHAIzmOG5Jj0v7Mlz2ogu-X_V-O3JNM6
+"""
+
+print("LANGUAGE CODE\nEnglish (en)\nTurkish (tr)\nAzerbaijani (az)\nIrish (ga)\nHindi  (hi)\nChinese   (zh)\nGreek  (el)\nJapanese  (ja)\nThai  (th)\nSpanish  (es)\nRussian  (ru)ETC...")
+lang=input(print("ENTER THE SELECTED LANGUAGE CODE : ", ))
+word=input(print("ENTER WORD FROM THE SELECTED LANGUAGE : ", ))
+print("THE GIVEN WORD IN LOWERCASE IS : ")
+
+p=[]
+if lang=='tr' or lang=='az':
+    for i in word:
+        if i=='I':
+           p.append('\u0131')
+        elif i>='A' and i<='Z':
+            p.append(i.lower())
+elif lang=='ga' or lang=='ga-IE':
+    if((word[0]=='n' or word[0]=='t') and (word[1]=='A' or word[1]=='E' or word[1]=='I' or word[1]=='O' or word[1]=='U' or word[1]=='\u00C1' or word[1]=='\u00C9' or word[1]=='\u00CD' or word[1]=='\u00D3' or word[1]=='\u00DA')):
+        p.append(word[0])
+        p.append('-')
+        if(word[1]=='\u00C1' ):
+            p.append('\u00E1')
+        elif(word[1]=='\u00C9'):
+            p.append('\u00E9')
+        elif(word[1]=='\u00CD'):
+            p.append('\u00ED')
+        elif(word[1]=='\u00D3'):
+            p.append('\u00F3')
+        elif(word[1]=='\u00DA'):
+            p.append('\u00FA')
+        elif(word[1]>='A' or word[1]=='E' or word[1]=='I' or word[1]=='O' or word[1]=='U' ):
+            p.append(word[1].lower())
+        c=word[2:]
+        for i in c:
+            if i>='A' and i<='Z':
+                p.append(i.lower())
+            elif(i=='\u00C1'):
+                p.append('\u00E1')
+            elif(i=='\u00C9'):
+                p.append('\u00E9')
+            elif(i=='\u00CD'):
+                p.append('\u00ED')
+            elif(i=='\u00D3'):
+                p.append('\u00F3')
+            elif(i=='\u00DA'):
+                p.append('\u00FA')
+            else:
+                p.append(i)
+    else:
+        for i in word:
```

Comment:
Great! I can see that this is a loop statement. Maybe we can have a line comment on this.